lib/xcodeproj/plist_helper.rb

```diff
@@ -70,6 +76,24 @@ def file_in_conflict?(path)
       ensure
         file.close
       end
+
+      def ruby_hash_write_xcode(hash, path)
```

Comment:
@neonichu Could you add some documentation to this stating which type this method expects. I.e, `path` being a pathname or string etc.